addons/html_builder/static/src/plugins/google_maps_option/google_maps_api_key_dialog.js

```diff
@@ -7,11 +7,11 @@ import { Component, useState, useRef } from "@odoo/owl";
  * @typedef {import('./google_map_option_plugin.js').ApiKeyValidation} ApiKeyValidation
```

Comment:
I made this commit separately to be able to revert it, in case. But it drives me crazy that in most places Google Maps was names Google Map :-P However, I kept the class name the same ("s_google_map") to avoid confusion.